docs/source/guide/testcase.rst

```diff
@@ -230,7 +154,7 @@ Adding a tag via Test Case view
 Removing a tag via Test Case view
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-#. Open the Test Case to be edited, click the **Tags** tab.
+#. Open the Test Case to be edited, navigate to **Tags** panel and input a tag.
```

Comment:
These "panel"'s are called cards in Patternfly speak and we've used 'cards' in changelog and commit logs so please change.